src/plugins/postgres_table_init_operator.py

```diff
@@ -48,8 +50,9 @@ def execute(self, context):
 
         # Define the SQL to execute by the super class.
         # This supports executing multiple statements in a single transaction:
+        init_operation = "DROP TABLE IF EXISTS" if self.drop_table else "TRUNCATE TABLE"
```

Comment:
Vraagje: moet tabel ververst worden in het geval van Dag restart?

Reply:
In principe wordt de tabel nu al gedropped/getruncate bij de rename aan het einde van de DAG.
Maar, net met Bart afgesproken een iets andere aanpak te gaan proberen. Namelijk de definitieve tabel altijd in leven te houden (ivm. de logical replication) en in Airflow die tabel te truncaten en dan via een copy operatie inlezen data (in de tmp) tabel, erin te kopieren (dus geen rename meer doen).